test/node.js

```diff
@@ -179,11 +185,15 @@ describe('filter addrs', () => {
     const mh2 = multiaddr('/ip4/127.0.0.1/udp/9090')
     const mh3 = multiaddr('/ip4/127.0.0.1/tcp/9090/ws')
     const mh4 = multiaddr('/ip4/127.0.0.1/tcp/9090/ws/ipfs/Qmb6owHp6eaWArVbcJJbQSyifyJBttMMjYV76N2hMbf5Vw')
+    const mh5 = multiaddr('/dns/ipfs.io/ws')
+    const mh6 = multiaddr('/dns/ipfs.io/ws/ipfs/Qmb6owHp6eaWArVbcJJbQSyifyJBttMMjYV76N2hMbf5Vw')
```

Comment:
Turns out that we apparently don't support ports in DNS address in js-mafmt, here is a PR that addresses it - https://github.com/whyrusleeping/js-mafmt/pull/15. I'm not entirely sure its the right approach here, but it seems to work.